### DISCUSSION TITLE
Can't list and access to Virtual machines

### DISCUSSION THREAD

**Discussion by nyevess** [netbox-community/netbox#15402] [Help Wanted!]
Hello, 
After upgrade my Netbox version to 3.7.2 I have the follow error when I'm try to access to the Virtual Machines list or one VM:
![image](https://github.com/netbox-community/netbox/assets/69692460/50669864-ab35-477b-bdee-9ceac3111a96)

Could you help me?

Thanks!!

**Comment by nyevess** [comment]
wrong answer

**Comment by candlerb** [comment]
You need to get a full backtrace, but I would suspect that the `slurpit_netbox` plugin hasn't been updated for Netbox 3.7.

Try temporarily commenting it out from your `PLUGINS` list in configuration.py, and see if the problem goes away.

**Comment by nyevess** [comment]
Hello @candlerb 
I tried to unistall the plugin following the netbox guide and the error continue, any suggestions?
![image](https://github.com/netbox-community/netbox/assets/69692460/b5cdc440-87fb-4a75-b977-e3b2399cb1c9)

**Reply by candlerb** [reply]
Didn't need to uninstall, just remove from `PLUGINS` as I said.

If your system is still broken, then you broke something during the upgrade - I'm afraid I can't tell what but you likely didn't follow the upgrade instructions properly. Repeat the upgrade process and look for errors, or make a fresh install and copy your database over to it.
